test_schemas/en/test_new_routing_case_insensitive_text_field.json

```diff
@@ -64,33 +64,31 @@
                             },
```

Comment:
Some text in the schema is now redundant and should be changed, it references terms like `Equals Any` which is now not the case, so should be updated with the new changes.

Reply:
updated description